internal/cobraprompt/prompt.go

```diff
@@ -134,7 +176,8 @@ func findSuggestions(co *CobraPrompt, d *prompt.Document) []prompt.Suggest {
 			suggestions = append(suggestions, prompt.Suggest{Text: flagUsage, Description: flag.Usage})
 		} else if (co.SuggestFlagsWithoutDash && d.GetWordBeforeCursor() == "") || strings.HasPrefix(d.GetWordBeforeCursor(), "-") {
 			if flag.Shorthand != "" {
-				flagUsage = fmt.Sprintf("-%s, %s", flag.Shorthand, flagUsage)
+				suggestions = append(suggestions, prompt.Suggest{Text: "-" + flag.Shorthand, Description: "or " + flagUsage + " " + flag.Usage})
```

Comment:
How about `fmt.Sprintf` instead of string concatenation?

Reply:
I thought this was more readable but sure https://github.com/hazelcast/hazelcast-commandline-client/pull/35/commits/bdd565a3b18e25cbb9e967db21a395523885a7e1